GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -150,10 +150,12 @@ public class NuGetUpgrader : ProductUpgrader
         /// <param name="src">The unprocessed string to use as arguments to an install command</param>
         /// <param name="installationId">A unique installer ID to replace the installer_id token with.</param>
         /// <returns>The argument string with tokens replaced.</returns>
-        public static string ReplaceArgTokens(string src, string installationId, string logsDirectory)
+        public static string ReplaceArgTokens(string src, string installationId, string logsDirectory, string installerBaseDirectory)
         {
-            string dst = src.Replace(NuGetUpgrader.replacementToken(InstallActionInfo.ManifestEntryLogDirectoryToken), logsDirectory)
-                .Replace(NuGetUpgrader.replacementToken(InstallActionInfo.ManifestEntryInstallationIdToken), installationId);
+            string dst = src
+                .Replace(NuGetUpgrader.replacementToken(InstallActionInfo.ManifestEntryLogDirectoryToken), logsDirectory)
+                .Replace(NuGetUpgrader.replacementToken(InstallActionInfo.ManifestEntryInstallationIdToken), installationId)

```

Comment:
Not your change, but why is `NuGetUpgader.replacementToken` in camelCase and not PascalCase?

Reply:
I was the one who originally wrote this - it should be PascalCased. I will fix this up in a separate commit (but just include it with this PR)